model/concourse.py

```diff
@@ -455,11 +462,31 @@ def expose_deployed_pipelines(self) -> bool:
         '''
         return self.raw.get('expose_pipelines', True)
 
-    def trusted_teams(self) -> typing.Iterable[str]:
+    def trusted_teams(self) -> collections.abc.Iterable[TrustedTeams]:
         '''Pull requests created/synchronized by members of this team will automatically have
         the required label set by the webhook-dispatcher
         '''
-        return self.raw.get('trusted_teams', [])
+        return [
+            dacite.from_dict(
```

Comment:
can we keep this simpler and just keep a list of strings? it should be fairly simple to discriminate between qualified and non-qualified teams, as github does not allow `/` (or `.` for that matter) characters in team-names.

Hence, entries of the form `foo/bar` (two parts if splitting by `/`) should be safe to translate to `<org>/<teamname>`, whereas entries of the form `host.tld/foo/bar` (three parts if splitting by `/`) should be safe to interpret as qualified